autoload/exconfig.vim

```diff
@@ -106,7 +106,7 @@ function exconfig#apply()
         let &tagrelative=0 " set notagrelative
 
         let s:old_tags=&tags
-        let &tags=fnameescape(g:exvim_folder.'/tags')
+        set tags+=fnameescape(g:exvim_folder.'/tags')
```

Comment:
I've already tested it. It seems OK to my Ubuntu with vim74, all the tags now can be added to &tags. And,  the &tags won't increase when save  the .exvim file or do :Update for the project.